lib/components/ContentModal/ContentModal.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { usePortal } from '@/helpers';
+import Chevron from '@/components/Chevron/Chevron';
+import styles from './ContentModal.css';
+
+const icons = require.context('./assets', true, /\.svg$/);
+
+export const ICONS = {
+  ATTACHMENT: 'attachment',
+  DRIVE: 'drive',
+  EMAIL: 'email',
+  REVOKE: 'revoke',
+  PROTECTED_MD: 'protected_md',
+  PROTECTED_TXT: 'protected_txt',
+  PROTECTED_JPEG: 'protected_jpeg',
+  PROTECTED_JPG: 'protected_jpg',
+  PROTECTED_PNG: 'protected_png',
+  PROTECTED_PPTX: 'protected_pptx',
+  PROTECTED_XLSX: 'protected_xlsx',
+  PROTECTED_DOCX: 'protected_docx',
+  PROTECTED_PDF: 'protected_pdf',
+  PROTECTED_EXT: 'protected_ext',
+  MD: 'md',
+  TXT: 'txt',
+  JPEG: 'jpeg',
+  JPG: 'jpg',
+  PNG: 'png',
+  PPTX: 'pptx',
+  XLSX: 'xlsx',
+  DOCX: 'docx',
+  PDF: 'pdf',
+  GENERIC: 'generic',
+  ERROR: 'error',
+};
+
+const getIcon = (alias) =>
```

Comment:
This makes me think maybe we should have an `Icon` component or similar, because this wouldn't necessarily be exclusive to this modal right?

Reply:
good point